src/library-authoring/common/context/ComponentPickerContext.tsx

```diff
@@ -8,7 +8,7 @@ import {
 
 export interface SelectedComponent {
   usageKey: string;
-  blockType: string;
+  blockType?: string;
```

Comment:
How does a Component ever have a `usageKey` but not a `blockType` ? Or is this really an ItemPicker that can select non-components?

Reply:
I was planning not to pass blockType when selecting 'Units', but I ended up passing it as the `containerType` has a similar meaning. Reverted 0962eabcf8fbbbb841cd0db3bb179f954ab8ea4a